src/main/java/io/outbound/sdk/PushNotification.java

```diff
@@ -348,13 +351,14 @@ public boolean wasMainActivityLaunched() {
 
         try {
             Uri media = null;
-            if (this.getSoundSilent().equals(true)){
-            } else if (this.getSoundDefault().equals(true)) {
-                media = RingtoneManager.getDefaultUri(RingtoneManager.TYPE_NOTIFICATION);
-            } else {
-                int resId = resources.getIdentifier(this.getSoundFile(), this.getSoundFolder(), context.getPackageName());
-                if (resId != 0) {
-                    media = Uri.parse("android.resource://" + context.getPackageName() + "/" + resId);
+            if (!this.getSoundSilent().equals(true)) {
```

Comment:
What does `getSoundSilent()` return? This looks like we're calling a method that returns a boolean, and doing a `.equals` on that, which seems odd to me. Am I missing something here? I see there's a bit of stringly-typed stuff going on, is that part of it?

Reply:
Good catch, I'll change that.